Projects/Checkers/Player.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Checkers;
+
+public class Player
+{
+	public bool IsHuman { get; private set; }
```

Comment:
Don't require setters here

Reply:
fixed in 880198f8088f584ce70c31b053d68bdbe5bd576b